pkgs/development/python-modules/bcrypt/3_1.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, isPyPy, fetchPypi
+, cffi, pycparser, mock, pytest, py, six }:
+
+with stdenv.lib;
+
+buildPythonPackage rec {
+  version = "3.1.7";
+  pname = "bcrypt";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0b0069c752ec14172c5f78208f1863d7ad6755a6fae6fe76ec2c80d13be41e42";
+  };
+  buildInputs = [ pycparser mock pytest py ];
```

```suggestion
  checkInputs = [ pytestCheckHook ];
```
I don't see any reference to ``pycparser, mock, py`` in the ``requirements.txt`` or ``setup.py``, so these are probably unnecessary.